intelligent-video-recording/config_detection.cfg

```diff
@@ -0,0 +1,15 @@
+# Use absolute path for model location and output file
+# Only MP4 recording is supported by the demo
+[DEMO_SETTINGS]
+CAMERA_DEVICE = /dev/video0
+CAMERA_DEVICE_WIDTH = 1280
+CAMERA_DEVICE_HEIGHT = 720
+ARCH=mobilenetv2ssd
+MODEL_LOCATION = /home/coral/models/ssd_mobilenet_v2_coco_quant_postprocess_edgetpu.tflite
+INPUT_LAYER = 'input'
+OUTPUT_LAYER = 'MobilenetV2/Predictions/Reshape_1'
```

Comment:
Please double check if the input and output names are needed in the coral. If not, please leave blank and leave a comment saying this is not needed for the coral, but other backends (such as TensorFlow) may need it.